rivet.spec

```diff
@@ -1,7 +1,8 @@
-### RPM external rivet 2.4.0
+### RPM external rivet 2.5.2
 Source: http://cern.ch/service-spi/external/MCGenerators/distribution/rivet/rivet-%{realversion}-src.tgz
+#Source: http://www.hepforge.org/archive/rivet/Rivet-%{realversion}.tar.gz  
 
-Requires: hepmc boost fastjet gsl yaml-cpp yoda
+Requires: hepmc fastjet gsl yoda
```

Comment:
Hi, there is nothing in the rivet and yoda toolfiles about these dependencies. Maybe it's missing since ever and I am not sure what has to go there. 

PS: I am cleaning the rest and will redo the PR in IB/CMSSW_8_1_X/gcc530.